src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -369,6 +369,28 @@ internal override (CSharpAttributeData?, BoundAttribute?) EarlyDecodeWellKnownAt
                     // diagnostics that might later get thrown away as possible when binding method calls.
                     return (null, null);
                 }
+                else if (CSharpAttributeData.IsTargetEarlyAttribute(arguments.AttributeType, arguments.AttributeSyntax, AttributeDescription.OverloadResolutionPriorityAttribute))
+                {
+                    if (this is SourcePropertyAccessorSymbol or SourceEventAccessorSymbol)
```

Comment:
Technically speaking, I don't think any of these (except lambdas) are places where the attribute would be _ignored_: it may be rather pointless to put them there, but overload resolution _will_ run on operators, conversions, and local function invocations. Lambdas would be ignored though, so we can error there.

Reply:
Local functions and unary operators Yes, they do involve overload resolution, but overloads are not possible so an overload resolution attribute doesn't make sense. I thought we error on positions where the attribute will be ignored.

User-defined operators: I don't see where overload resolution is involved in the [spec](https://github.com/dotnet/csharpstandard/blob/draft-v8/standard/conversions.md#1054-user-defined-implicit-conversions) or the code.